packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/WebhookForm/tests/index.test.js

```diff
@@ -88,6 +89,7 @@ describe('Create Webhook', () => {
         events: ['entry.create'],
         headers: [{ key: '', value: '' }],
       });
+      expect(saveButton).toHaveAttribute('aria-disabled', 'true');
```

Comment:
I think this can be outside the `waitFor` at this point, we know the submit has been called?